### ISSUE TITLE
Typo in lint rule in analysis_options_user.yaml

### ISSUE THREAD

**Issue by zimmi** [flutter/flutter#12067]
It reads [unnecessary_brace_in_string_interp](https://github.com/flutter/flutter/blob/b91e8f97efb43057887ed8ffa890c0c2c3b7f672/packages/flutter/lib/analysis_options_user.yaml#L88) but should be [unnecessary_brace_in_string_interp**s**](http://dart-lang.github.io/linter/lints/unnecessary_brace_in_string_interps.html).

The Dart-Code plugin for VSCode warned me about it in my `analysis_options.yaml`.
It looks like this:
```yaml
include: package:flutter/analysis_options_user.yaml

# add some more strictness
analyzer:
  strong-mode:
    implicit-casts: false
    implicit-dynamic: false
```

**Comment by a14n** [comment]
Fixed by #12034

**Comment by zimmi** [comment]
Woops, youre're right. Should have searched more. Thanks!
